packages/flutter_tools/lib/src/usage.dart

```diff
@@ -44,35 +44,43 @@ class Usage {
   Analytics _analytics;
 
   bool _printedUsage = false;
+  bool _suppressAnalytics = false;
```

Comment:
The paranoid in me wants me to suggest that this should default to true and we should set it to false when we know the user has not opted out, instead of the other way around.